rest/endpoints/media/recording_playable_stream.yml

```
@@ -37,6 +37,13 @@ paths:
           required: true
           schema:
             type: string
+        - in: path

```

This parameter is not in the path
```suggestion
        - in: query
```